### DISCUSSION TITLE
Contentful data schemas and status of FEM for stand-alone installations

### DISCUSSION THREAD

**Discussion by hrdl-github** [zooniverse/front-end-monorepo#5275] [General]
I'm trying to set up a stand-alone panoptes installation. I haven't managed to set up the FEM yet, as it requires some data from Contentful. Do you provide the required data schemas? In general, as the FEM intended to be used already or should I use the legacy frontend for the time being?

**Comment by goplayoutside3** [comment]
Hi there! Apologies for the long delay. You should now be able to bootstrap FEM locally without Contentful tokens. 

FEM is being used for select Zooniverse projects while PFE (legacy frontend) is still used for almost everything else. All new features and new webpages are being built into FEM though. If you're curious about which projects use FEM, see PFE's router [utils file](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/monorepoUtils.js). 

We do welcome new PRs and Issues about FEM, but the dev team does not spend nearly as much time maintaining PFE.

**Reply by hrdl-github** [reply]
Thanks, I stopped getting the FEM to work a while ago. I'm still not too optimistic about replicating the entire zooniverse stack in a stand-alone setup. Is my understanding that this running all of https://github.com/zooniverse/panoptes, PFE, FEM, https://github.com/zooniverse/talk-api, https://github.com/zooniverse/sugar, and the stats service? I'm a bit weary of all the hard-coded hostnames that remain and which build type (production / development) would be most appropriate before I start modifying any source files. Are you aware of any existing stand-alone instances besides the official one?

**Reply by goplayoutside3** [reply]
We don't recommend running a stand-alone setup of the entirety of the Zooniverse website. Each of the repos mentioned have their own `staging` deploy and individual instructions for running locally.
